js/fullcalendar.js.php

```diff
@@ -269,13 +269,24 @@
 				}
 
 
-				if(event.isDarkColor == 1) {
-					element.css({ color : "#fff" });
-
-					element.find('a').css({
-						color:"#fff"
-					});
-				}
+				if(event.isDarkColor == 0) {
+					element.css({ color : "#000000" });
+
+                    element.find('a').find('span').each(function() {
```

Comment:
Je pense qu'on peut simplifier un peu (à vérifier) :
```suggestion
                    element.find('a span').each(function() {
```
Sinon, je crois que tu as indenté avec des espaces au lieu des tabs…